logfire/_internal/main.py

```diff
@@ -1948,13 +1948,13 @@ def __init__(self, span: trace_api.Span) -> None:
         self._token = context_api.attach(trace_api.set_span_in_context(self._span))
 
     def __enter__(self) -> FastLogfireSpan:
+        self._span.__enter__()
```

Comment:
This is maybe unnecessary but seems reasonable to add, and I hope this class will be refactored/simplified soon anyway.

Reply:
Given how optimised this class is, I'd rather not add it here, but maybe it can stay in LogfireSpan.